ang/civicase/DashboardTab.js

```diff
@@ -1,10 +1,109 @@
-(function (angular) {
+(function (angular, $, _) {
   var module = angular.module('civicase');
 
   module.directive('civicaseDashboardTab', function () {
     return {
       restrict: 'E',
+      controller: 'dashboardTabCtrl',
```

Comment:
Following the naming convention we have so far, should be `dashboardTabController`.